package.json

```diff
@@ -97,7 +97,6 @@
         }
       ],
       "no-constant-condition": 0,
-      "openlayers-internal/enum": 2,
```

Comment:
Instead of disabling this rule, you can follow it by giving public enums their own file.